### ISSUE TITLE
/api/queries/id does not give theory

### ISSUE THREAD

**Issue by lex-lex** [normativeai/backend#15]
When calling GET `api/queries/id`, the response object does not contain the `theory` entry, even when set during creation (that is POST `api/query` with an query object containing `theory`).

**Comment by shaolintl** [comment]
Did you check the data field in the response? I changed the whole response
object according to your request...

**Comment by lex-lex** [comment]
Example:
POST `api/queries/` with post data
gives as answer: 
```
{"data":{"assumptions":[""],"_id":"5c3e5f50972c9a60bae2f0cb","name":"New Query","lastUpdate":"2019-01-15T22:31:44.016Z","user":"5c3dbef15d2d331e567dede0","theory":"5c3dbefb5d2d331e567dede1","description":"","goal":"","__v":0}}
```
(so all according to plan).

However, now calling GET `api/queries/5c3e5f50972c9a60bae2f0cb` gives me

```
{"data":{"assumptions":[""],"_id":"5c3e5f50972c9a60bae2f0cb","name":"New Query","lastUpdate":"2019-01-15T22:31:44.016Z","description":"","goal":""}}
```
(note no theory field)

**Comment by shaolintl** [comment]
Sorry, I didnt read the message attentively. Should work find now.

**Comment by lex-lex** [comment]
As far as I understand from the code, now not only the theory id is contained in `query.theory` but instead the whole theory object. Is this right?

**Comment by shaolintl** [comment]
No, not everything but the name, description and last update. This can be
easily changed if you like.

**Comment by shaolintl** [comment]
Yes, a mistake. I’ll remove the assumptions and leave only name and description.

**Comment by lex-lex** [comment]
It's just that previously you said that `query.theory` is the id (string) of the associated theory. Do I need to provide a "theory object" for saving a query or just the id string?
E.g.
```
{_id: "5c3e5f50972c9a60bae2f0cb", name: "New Query", theory: "5c3dbefb5d2d331e567dede1",…}
```
vs.
```
{_id: "5c3e5f50972c9a60bae2f0cb", name: "New Query", theory:  { ... theory object ....},…}
```
as payload for POST `api/queries/id` for updating a query.
Currently, it seems that the API accepts a string for POST but gives an object for GET.

**Comment by shaolintl** [comment]
Yes, it is enough to have the id for saving but I return a bit more on
getting. Seems logical to me but can easily be changed.
